crates/uv-installer/src/plan.rs

```diff
@@ -305,14 +307,16 @@ impl<'a> Planner<'a> {
                     install_path,
                     lock_path,
                 } => {
-                    // Store the canonicalized path, which also serves to validate that it exists.
-                    let install_path = match install_path.canonicalize() {
-                        Ok(path) => path,
-                        Err(err) if err.kind() == std::io::ErrorKind::NotFound => {
-                            return Err(Error::NotFound(url.to_url()).into());
-                        }
-                        Err(err) => return Err(err.into()),
-                    };
+                    // Convert to an absolute path.
+                    let install_path = std::path::absolute(install_path)?;
+
+                    // Normalize the path.
+                    let install_path = normalize_absolute_path(&install_path)?;
```

Comment:
Should we have a helper for this like `user_display`? that was really helpful for usage.

Reply:
Yeah something like... `absolutize`?

Reply:
I'm going to do it in a different PR because I want to remove our `absolutize` dependency.